app/src/main/java/com/onegini/mobile/exampleapp/view/activity/AuthenticationActivity.java

```diff
@@ -110,6 +115,21 @@ private boolean isBlank(final String string) {
 
   @Override
   public void onBackPressed() {
-    // we don't want to be able to go back from the pin screen
+    if(isCancellable()){
+      super.onBackPressed();
```

Comment:
We must also handle this `super.onBackPressed()` in the classes that inherit from this `AuthenticationActivity`. 

E.g. in the `PinActivity` class we must add the following snippet in order to actually cancel the authentication request:
```
  @Override
  public void onBackPressed() {
    onCancelClicked();
  }
```